Post/views.py

```diff
@@ -11,8 +12,20 @@ class PostListView(ListView):
     context_object_name = 'posts'
 
 
+@login_required
+def CreateNewPost(request):
+    form = CreatePostForm(request.POST)
+    if form.is_valid():
+        form.save()
```

Comment:
Maybe add here a success messages that the post has created. (Inside the if block)
You can do so by adding `messages.success(request, 'The post has been created' )`

Reply:
@Leeadar, Nice! 
I added the message.